platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -1,3 +1,16 @@
+{#-
```

Comment:
@keboliu is this the fixed version I remember we had an issue with it and a PR that fixes it. Should we have the fixed PR on top of this one or fixed it already in this PR?

Reply:
I combined the fix to this PR, and that PR will NOT be needed for 202305, only for 202211.